File: myapp/src/app/product.ts
export class Product{
    productId: string;
    brand: string;
    descrption: string;
    quantity: number;
    price: number;

    constructor(productId: string, brand: string, description: string, quantity: number, price: number){
        this.productId=productId;
        this.brand=brand;
        this.descrption=description;
        this.quantity=quantity;
        this.price=price
    }
}

File: Spring_projects/firstRestJPA_MySQL/src/main/java/com/iiht/restJPA/MYSQL/entity/Product.java
package com.iiht.restJPA.MYSQL.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
public class Product {
	
	@Id
	@GeneratedValue(strategy=GenerationType.IDENTITY)
	private long productId;
	private String description;
	private double price; 
	private String brand;
	private long quantity;

}
